src/core_codemods/fix_empty_sequence_comparison.py

```diff
@@ -0,0 +1,67 @@
+import libcst as cst
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin, AncestorPatternsMixin
+
+
+class FixEmptySequenceComparison(
+    BaseCodemod, NameResolutionMixin, AncestorPatternsMixin
+):
+    NAME = "fix-empty-sequence-comparison"
+    SUMMARY = "Replace Comparisons to Empty Sequence with Implicit Boolean Comparison"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
```

Comment:
This needs to be `MERGE_AFTER_REVIEW` since this has the potential to break existing code.